.circleci/config.yml

```diff
@@ -1,5 +1,17 @@
 version: 2
 jobs:
+  build:
+    docker:
+      - image: circleci/node:latest
+    steps:
+      - checkout
+      - setup_remote_docker
+      - run: npm install
+      - run: 
+          name: Docker compose up
```

Comment:
This looks right to me! Would it be worth adding something to test that the containers stayed up? Maybe a `curl localhost:3000` to make sure the registry responds, or something like that?